nextcloud_install_production.sh

```diff
@@ -704,9 +704,14 @@ rm -f *.zip.*
 done;
 
 # Install virtual kernels
-apt install linux-tools-virtual-hwe-16.04-edge linux-cloud-tools-virtual-hwe-16.04-edge  -y
-apt install linux-image-virtual-hwe-16.04-edge -y
-apt install linux-virtual-hwe-16.04-edge -y
+apt purge -y $(dpkg-query -W -f='${binary:Package}\n' 'linux-image-4*' 'linux-headers-4*' | grep -v 4.4.0-64-generic)
```

Comment:
IIRC, Linux 4,8 is like Ubuntu 16.10 as 4.4 like 16.04.
Linux 4.4 is a LTS release.